pkg/logutil/zap_log_test.go

```diff
@@ -153,7 +155,8 @@ func (t *testZapLogSuite) TestLog(c *C) {
 			`[object2={username="user 2"}] [binary="YWIxMjM="] ["is processed"=true] `+
```

Comment:
This encoding looks incorrect, as there is a `"` in the `{username="user 2"}`.

As I talked before, I think the best solution is to *always* check whether its string representation has special characters.